recipes/configs/mistral/7B_full_mps.yaml

```
@@ -0,0 +1,70 @@
+# Config for single-device M based Mac full finetuning using a Mistral 7B model
+# 
+# This config uses hyperparameters based on small set of experiments and information
+# available on various forums. These are not meant to replicate the numbers
+# from the paper
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#   tune download mistralai/Mistral-7B-v0.1 --output-dir ./tmp/Mistral-7B-v0.1
+#
+# You can then run the recipe with:
+#   tune run full_finetune_single_device --config ./recipes/configs/mistral/7B_full_mps.yaml
+# Tokenizer
+tokenizer:
+  _component_: torchtune.models.mistral.mistral_tokenizer
+  path: ./tmp/Mistral-7B-v0.1/tokenizer.model
```

```suggestion
  path: /tmp/Mistral-7B-v0.1/tokenizer.model
```



nit: we actually want to download to the real `/tmp` directory in our examples so as to not persist a bunch of models on disk. Currently, this will write to `$CWD/tmp/Mistral-7B-v0.1`. This should be changed in a few places in the config.

---

on mac /tmp resolves to /private/tmp and it actually breaks the symlinks that are created, that's why I ended up doing this :(

---

Gotcha, then I would actually prefer something like `./models/` directory so we don't confuse this with an actual tmp dir.